src/Tribe/PUE/Notices.php

```diff
@@ -212,6 +212,15 @@ public function render_invalid_key() {
 			return;
 		}
 
+		// Only show our invalid key messaging on the plugin admin screen and on our own admin screens
+		// @todo review and revise in MR 17.13
+		if (
+			'plugins.php' !== $pagenow
```

Comment:
This is consistent with the preceding lines and is temporary; can explore it if you think it's really better though